docs/changelog.rst

```diff
@@ -7,34 +7,46 @@ Changelog
 Next Version
 ------------
 
-Logging configurable in the deployed entity service by using the key ``loggingCfg``.
 
 
-Several old settings have been removed from the default values.yaml and docker files:
-- ``SMALL_COMPARISON_CHUNK_SIZE``
-- ``LARGE_COMPARISON_CHUNK_SIZE``
-- ``SMALL_JOB_SIZE``
-- ``LARGE_JOB_SIZE``
 
-Have all been replaced by ``CHUNK_SIZE_AIM``
-
-``MATCH_THRESHOLD`` -> ``ENTITY_MATCH_THRESHOLD``
-But really should be removed in favour of each run providing a threshold.
-
-Celery configuration updates to solve threads and memory leaks in deployment.
-Update helm charts and docker-compose to use these new preferred configurations.
-
-Update helm `values.yaml` file to be a minimal working deployment.
-
-New environment variables: ``CELERY_DB_MIN_CONNECTIONS``, ``FLASK_DB_MIN_CONNECTIONS``, ``CELERY_DB_MAX_CONNECTIONS`` and ``FLASK_DB_MAX_CONNECTIONS``
-to configure the database connections pool.
-
-
-Update benchmark to handle more than 2 parties, being able to repeat experiments and pushing the results to minio object store.
-Required a change in the result created by the benchmark.
-
-Deleting a run is now implemented.
```

Comment:
This is in the 1.12 milestone so should still be included in the changelog notes?
 https://github.com/data61/anonlink-entity-service/milestone/19?closed=1

Reply:
See line 30. I just moved everything on a probably random fashion without proper reasoning. :)